sdcm/cluster.py

```diff
@@ -1356,6 +1356,10 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster',
         self.nemesis = []
         self.params = params
         self.add_nodes(n_nodes)
+        for node in self.nodes:
+            status = node.remoter.run('sudo systemctl status scylla-server.service', ignore_status=True)
+            if 'Active: inactive (dead)' in status.stdout:
+                self.fail("some nodes failed to start")
```

Comment:
self.fail() doesn't exist.

We can't assume that nodes has a running scylla-server service after being added to cluster.
We might add a clean/blank node and install and setup scylla in `setup` stage.

This commit blocked all GCE tests, let's revert it?